src/components/campaigns/update-form/index.tsx

```
@@ -0,0 +1,847 @@
+                      {safeFormatDate(campaignEndDate)}
+                    </dd>
+                  </div>
+                  <div className="col-span-full">
+                    <dt className="text-sm text-muted-foreground">
+                      Description
+                    </dt>
+                    <dd className="whitespace-pre-wrap text-sm font-medium">
+                      {watch("description") ? (
+                        <div
+                          className="ql-editor preview"
+                          dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)